spec/unit/classes/database/ssl_configuration_spec.rb

```diff
@@ -184,7 +187,7 @@
       it 'has read ident rule' do
         is_expected.to contain_postgresql__server__pg_ident_rule("Map the SSL certificate of the server as a #{params[:read_database_username]} user")
           .with_map_name(read_identity_map)
-          .with_system_username(facts[:fqdn])
+          .with_system_username(facts[:networking][:fqdn])
```

Comment:
shouldn't that be:
```suggestion
          .with_system_username(facts[:networking]['fqdn'])
```

Reply:
Humm… it seems to be worse.  That being said at the beginning of the file we have:

```ruby
      let(:facts) do
        {
          # ...
          networking: {
            fqdn: 'cheery-rime@puppet',
          },
          # ...
        }
      end
```

This module is hell… :scream: